source/blog/2017-08-16-ovirt-ansible-roles-how-to-use.html.md

```diff
@@ -0,0 +1,149 @@
+---
+title: How to use oVirt Ansible roles
+author: omachace
+tags: Ansible, Roles, Automation, Configuration management
+date: 2017-08-16 15:00:00 CET
+comments: true
+published: true
+---
+
+In blog post [An Introduction to oVirt Ansible Roles](./2017-07-19-ovirt-ansible-roles-an-introduction.html.md) you can have read about new roles introduced in latest oVirt release. This blog post is follow up to show you how to set it up and use it.
+
+## Ansible Galaxy
+
+You may be familiar with [Ansible galaxy](https://galaxy.ansible.com/). Ansible galaxy stores many different Ansible roles, which may simplify your life. We store [oVirt Ansible roles](https://galaxy.ansible.com/oVirt/ovirt-ansible-roles/) in Ansible galaxy as well. In order to use oVirt roles from galaxy you need to perform following steps:
+
+```bash
+$ ansible-galaxy install ovirt.ovirt-ansible-roles
+```
+
+This will install your roles into directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/`. This directory contains many roles, so in order to use them you need to tell Ansible where to look for these roles, as Ansible by default search for roles only in `/etc/ansible/roles/` and your current working directory. In order to change the directories where Ansible look for roles, you need to modify `roles_path` option of `[defaults]` section in `ansible.cfg` configuration file. The default location of this file is in `/etc/ansible/ansible.cfg`. More information can be found on [Ansible documentation](http://docs.ansible.com/ansible/latest/intro_configuration.html) pages.
+
+```bash
+$ sed -i 's|#roles_path    = /etc/ansible/roles|roles_path = /etc/ansible/roles:/etc/ansible/roles/ovirt.ovirt-ansible-roles/roles|'  /etc/ansible/ansible.cfg
+```
+
+Now copy one of the examples from directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/examples/` into your working directory modify the needed variables and run the playbook.
```

Comment:
Suggest:
"Copy one of the examples from the directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/examples/` into your working directory, then modify the needed variables and run the playbook."